### PR TITLE
Merge and PCA of the pooled watermelon count matrix and the pooled wa…

### PR DESCRIPTION
**Pull request by quinn-mccall**
…termelon data matrix.

### CODE REVIEW

**Review by aarmey — CHANGES_REQUESTED** [review]
Can you please make both of these Python functions, and then move them to `imports.py`?

**Comment by quinn-mccall** [comment]
Hi Dr. Meyer! I just added the function to imports.py and called it and wrote assertions for it in tests.py, let me know what you think!

File: pf2barcode/imports.py [hunk 1]
```diff
@@ -72,3 +72,21 @@ def import_CCLE() -> anndata.AnnData:
...
+        "/opt/extra-storage/GSE150949/GSE150949_metaData_with_lineage.txt.gz",
+        delimiter="\t",
+        engine="python",
+    )
+    # separate the columns to merge with the data
+    columns = metadata[["full_cell_barcode", "lineage_barcode"]]
+    # create anndata object of the data file
+    data = anndata.read_csv(data_file, delimiter=",")
+    # merge the data file object with the metadata coluns
+    data.obs = data.obs.join(columns, how="left")
```
Comment (aarmey) [inline pf2barcode/imports.py:89]: After this line, could you use `data.write("filename")` with each file to create an anndata file of each dataset? This long importing will quickly get tiring, so we should just store the datasets as an anndata file. If you save them on your account on Aretha, I can then move them somewhere where everyone can access.

**Review by aarmey — CHANGES_REQUESTED** [review]
Thanks @quinn-mccall! I made some small adjustments, including updating the file paths so that this runs on Aretha. Please see my comment about speeding this up.

**Review by aarmey — APPROVED** [review]
@quinn-mccall this did turn out to be trickier than I expected. I have set up the files, and you should find that this is much faster now.

**Pull request merged**